server/routes/alumni.js

```diff
@@ -20,9 +20,26 @@ const opportunitySchema = require('../models/opportunitySchema');
 var sendAlumniVerificationEmail = require('../routes/helpers/emailHelpers').sendAlumniVerificationEmail;
 const { ObjectId } = require('mongodb');
 require('mongoose').Promise = global.Promise
+var FOOTY_POINTS_CHART = require('../footyPointsChart').FOOTY_POINTS_CHART
 
 const HASH_COST = 10;
 
+/**
+ * Finds the alumnus or student account by the id of the referrer provided and awards footyPoints
+ * @param {*} referrerId 
+ */
+const awardReferralPoints = async (referrerId) => {
```

Comment:
nit: it might be worth having a footyPoint helper file or something along those lines. Because as the concept grows, figuring out where methods like this belong may be a little harder.